code/nx.json

```diff
@@ -1,10 +1,29 @@
 {
-  "$schema": "./node_modules/nx/schemas/nx-schema.json",
-  "pluginsConfig": {
-    "@nrwl/js": {
-      "analyzeSourceFiles": false
-    }
+  "$schema": "../scripts/node_modules/nx/schemas/nx-schema.json",
+  "extends": "nx/presets/npm.json",
+  "nxCloudAccessToken": "NGVmYTkxMmItYzY3OS00MjkxLTk1ZDktZDFmYTFmNmVlNGY4fHJlYWQ=",
```

Comment:
Shouldn't we use CI variables here: https://nx.dev/ci/recipes/security/access-tokens#setting-a-different-access-token-in-ci?

If so, make sure to regenerate the token, since this is now publicly available.

Reply:
This is a readonly one, that they recommend to put in GIT.
TBH, all core maintainers should add the read-write token to their environment, so that if someone already build the packages locally, than CI or other maintainers can get it from cache.